src/components/dls/Separator/Separator.module.scss

```diff
@@ -0,0 +1,15 @@
+$separator-size: 1px;
+
+.base {
+  background-color: var(--color-borders-hairline);
+
+  &[data-orientation="horizontal"] {
+    height: $separator-size;
+    width: 100%;
+  }
+
+  &[data-orientation="vertical"] {
+    height: 100%;
```

Comment:
height: 100% doesn't actually take 100% height. It takes 100% of the width. That's a limitation of the % usage in css. 

There's another css property for the container height, not sure what it was

Reply:
would love to learn more about this.